course_discovery/apps/course_metadata/models.py

```diff
@@ -588,6 +588,8 @@ class CourseRun(TimeStampedModel):
         verbose_name=_('Add OFAC restriction text to the FAQ section of the Marketing site')
     )
 
+    invite_only = models.BooleanField(default=False)
```

Comment:
Any specific reason to not enforce not null?

Reply:
This information is sent in the API response. doesnt make sense to send a null when we are checking it for boolean value. this will be set to false anyways upon running `refresh_course_metadata`.

Reply:
Not null just ensures false is always setting in value but I think adding default also enforce same thing so both way it's good.